source/documentation/modules/map.html.md.erb

```diff
@@ -28,78 +67,297 @@ title: sass:map
     ===
     $font-weights: ("regular": 400, "medium": 500, "bold": 700)
 
-    @debug map.has-key($font-weights, "regular")  // true
-    @debug map.has-key($font-weights, "bolder")  // false
+    @debug map.has-key($font-weights, "regular") // true
+    @debug map.has-key($font-weights, "bolder") // false
+  <% end %>
+
+  If `$keys` is not empty, the map targeted for searching is nested within
+  `$map`. From left to right, including `$key` and excluding the last key in
```

Comment:
Avoid passive voice here. Maybe instead: "If `$keys` is not empty, follows them from left to right to find a nested map to check." Using "follows" here mirrors your explanation of nested maps above. Same suggestion for other functions.